docs/developer-guide/coal-setup.md

```diff
@@ -100,96 +162,535 @@ the last step in configuration allows editing the resulting configuration file.
+mahi                                 running         -                   online
+sdc                                  running         -                   online
+papi                                 running         -                   online
+dhcpd                                running         -                   online
+redis                                running         -                   online
+ufds                                 running         -                   online
+manatee                              running         -                   online
+vmapi                                running         -                   online
+cnapi                                running         -                   online
+ca                                   running         -                   online
+global                               running         provisioner         online
+global                               running         heartbeat           online
+global                               running         ur                  online
+global                               running         smartlogin          online
+```
 
-![Login prompt on console.](../img/coal-headnode-login-prompt.png)
 
-- After sometime you will see, preparing for setup:
+### Additional Plumbing
+
+1. Add external nics to imgapi and adminui
+
+  These are required in order to be able to access remote update sources, and
+  in order to be able to access AdminUI using a browser:
+
+    ```bash
+    [root@headnode (coal-1) ~]# sdcadm post-setup common-external-nics
+    Added external nic to adminui
+    Added external nic to imgapi
+    ```
+
+  Please note that this command didn't wait for the "add nics" jobs to be
+  completed, just submitted, so you might need to give it some extra time
+  after the command exits until these jobs really finish.
+
+  Let's use "sdc-vmapi" to confirm which services have an external IP:
+
+    ```bash
+root@headnode (coal-1) ~]# sdc-vmapi /vms?state=running | json -H -ga alias nics.0.ip nics.1.ip
+dhcpd0 10.99.99.9
+imgapi0 10.99.99.21 10.88.88.4
+sdc0 10.99.99.28
+workflow0 10.99.99.19
+napi0 10.99.99.10
+fwapi0 10.99.99.26
+assets0 10.99.99.8
+moray0 10.99.99.17
+ufds0 10.99.99.18
+redis0 10.99.99.24
+sapi0 10.99.99.32
+vmapi0 10.99.99.27
+cnapi0 10.99.99.22
+binder0 10.99.99.11
+amonredis0 10.99.99.23
+rabbitmq0 10.99.99.20
+manatee0 10.99.99.16
+papi0 10.99.99.29
+amon0 10.99.99.25
+ca0 10.99.99.30
+mahi0 10.99.99.33
+adminui0 10.99.99.31 10.88.88.3
+```
 
-!["preparing for setup..." on console.](../img/coal-preparing-for-setup.png)
+  We can now access access the operations portal, "SDC ADMINUI", in a web
+  browser on the host computer at https://10.88.88.3/ .
 
-- Finally, when setup is complete you'll see:
+2. Set up CloudAPI
 
-!["Setup complete on console."](../img/coal-setup-complete.png)
+  CloudAPI provides the self-serve API access to SDC. If you are developing or
```

Comment:
Excellent point. I don't want to add anything docker here while docker docs are confined to joyent/sdc-docker during this development cycle.